node/src/reactor/joiner.rs

```diff
@@ -352,6 +369,7 @@ pub(crate) struct Reactor {
     block_validator: BlockValidator<NodeId>,
     deploy_fetcher: Fetcher<Deploy>,
     linear_chain: linear_chain::LinearChainComponent<NodeId>,
+    console: Console,
```

Comment:
Can we find a better name for it? `console` is too ambiguous. Maybe `DiagnosticConsole` or `DebuggingConsole`?

Reply:
There shall only be one! I'm loathe to change it at this point, to be honest. If you strongly feel it is necessary, I will though :)